explanation/performance/perf-pgo.md

```diff
@@ -0,0 +1,153 @@
+(perf-pgo)=
+
+# Profile-Guided Optimization
+
+[Profile-guided optimization](https://en.wikipedia.org/wiki/Profile-guided_optimization) (PGO) or Feedback-driven optimization (FDO) are synonyms for the same concept.  The idea is that a binary is compiled with built-in instrumentation which, when executed, will generate a profile.  This profile can then be used as an input for the compiler in a subsequent build of the same binary, and it will serve as guide for further optimizations.
```

Comment:
```suggestion
[Profile-guided optimization](https://en.wikipedia.org/wiki/Profile-guided_optimization) (PGO) or feedback-driven optimization (FDO) are synonyms for the same concept.  The idea is that a binary is compiled with built-in instrumentation which, when executed, will generate a profile.  This profile can be used as input for the compiler in a subsequent build of the same binary, and it will serve as a guide for further optimizations.
```